table/src/main/java/tech/ydb/table/Session.java

```diff
@@ -72,6 +74,12 @@ CompletableFuture<Result<DataQueryResult>> executeDataQuery(
         Params params,
         ExecuteDataQuerySettings settings);
 
+/**
+ *      Keys must be a list of structs where each struct is a primary key
```

Comment:
No more keys here :)
Comment should be moved to ReadRowsSettings

Reply:
Thanks